### PR TITLE
Adding tidb adapter

### PR DESCRIPTION
**Pull request by zhexuany**
Adding a plugin for datanucelus.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -120,6 +120,74 @@
...
+            <version>2.7.2</version>
+            <scope>${spark.scope}</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.pingcap.tidb</groupId>
+            <artifactId>tidb-datanucleus-adapter</artifactId>
+            <version>1.0</version>
+            <scope>compile</scope>
+        </dependency>
+    </dependencies>
```
Comment (marsishandsome) [inline pom.xml]: why add these dependencies?

Comment (zhexuany) [reply pom.xml]: In this way, we can use the plugin at runtime.

**Comment by zhexuany** [comment]
build

**Pull request merged**